migration-engine/migration-engine-tests/tests/native_types/mysql.rs

```diff
@@ -464,12 +464,12 @@ const IMPOSSIBLE_CASTS: Cases = &[
     ),
     (
         "Double",
-        quaint::Value::Float(Some(3.14)),
+        quaint::Value::Float(Some(3.20)),
         &["Binary(10)", "Date", "Timestamp(0)", "DateTime(0)"],
     ),
     (
         "Float",
-        quaint::Value::Float(Some(3.14)),
+        quaint::Value::Float(Some(3.20)),
```

Comment:
What is the reason for changing that?

Reply:
Clippy thinks we're trying to approximate pi here and suggests using the `PI` constant. Given we don't actually want Pi, but just a test input, it's easier to just change it.